src/org/camunda/latera/bss/utils/Order.groovy

```diff
@@ -18,9 +18,11 @@ import org.camunda.latera.bss.connectors.Minio
 
 class Order implements GroovyObject {
   private DelegateExecution _execution
+  Boolean _raw
 
-  Order(DelegateExecution execution) {
+  Order(Map params = [:], DelegateExecution execution) {
     this._execution = execution
+    this._raw = params.raw ?: execution.getVariable('rawOrderData') ?: false
```

Comment:
You get ```rawOrderData``` or ```false```?

Reply:
If `raw` named arg is passed, use it.
Otherwise try to use value of execution variable. In this case you can declare behavior for the whole business process instance instead of passing it into every constructor call.
If variable is not set, use default `false` value.

Reply:
Okay, describe me the type of ```this._raw```?

Reply:
Boolean

Reply:
Okay, give me example of data stored in ```rawOrderData```

Reply:
`true`/`false`